packages/xo-server/src/xo-mixins/plugins.js

```diff
@@ -203,7 +203,13 @@ export default class {
       throw invalidParameters('plugin not configured')
     }
 
+    if (plugin.loading) {
+      throw invalidParameters('plugin is loading')
```

Comment:
Why you threw `invalidParameters` error ? I don't see the relation between parameters and loading plugin !

Reply:
Indeed, but I used exactly the same way as the tests above.

I think the original idea was that the `id` param is invalid because this plugin is not in a state that can be used with this function.

I agree with you that using an `invalidState` error would make more sense but I did not want to introduce more changes at this time.